jwst/extract_2d/grisms.py

```diff
@@ -441,14 +442,14 @@ def extract_grism_objects(input_model,
                 order_model.inverse = Const1D(order)
 
                 tr = inwcs.get_transform('grism_detector', 'detector')
-
                 tr = Mapping((0, 1, 0, 0, 0)) | (Shift(xmin) & Shift(ymin) &
                                                  xcenter_model &
                                                  ycenter_model &
                                                  order_model) | tr
-                from gwcs.utils import _toindex
-                y_slice = slice(_toindex(ymin), _toindex(ymax))
-                x_slice = slice(_toindex(xmin), _toindex(xmax))
+
+                y_slice = slice(_toindex(ymin), _toindex(ymax) + 1)
+                x_slice = slice(_toindex(xmin), _toindex(xmax) + 1)
```

Comment:
Is there any worry with making both axes inclusive then running into a slice of e.g. [n:2049] in the dispersion direction, putting you off the detector? I may not be comprehending this change fully.

Reply:
That had occurred to me too, but theoretically it should not happen. The coords we get back from all the transforms are zero-indexed, so the highest that xmax/ymax should ever be is 2047 (or NAXISn - 1). So when we add 1 to end slice limit the max we should ever get is 2048, which is valid for a data array of size 2048.

Reply:
Interesting - I only ask because this asdf issue with WFSS data has shown some cutouts (in log outputs, at least) to be something like xmin: 1255, xmax: 2048, so I wasn't sure if we were already reaching the edge before this PR. I'll just fetch master and make sure nothing breaks. 🙂